Content.Server/Glue/GlueSystem.cs

```diff
@@ -21,55 +22,70 @@ public sealed class GlueSystem : SharedGlueSystem
     [Dependency] private readonly IGameTiming _timing = default!;
     [Dependency] private readonly MetaDataSystem _metaData = default!;
     [Dependency] private readonly IAdminLogManager _adminLogger = default!;
+    [Dependency] private readonly OpenableSystem _openable = default!;
 
     public override void Initialize()
     {
         base.Initialize();
 
         SubscribeLocalEvent<GlueComponent, AfterInteractEvent>(OnInteract, after: new[] { typeof(OpenableSystem) });
         SubscribeLocalEvent<GluedComponent, ComponentInit>(OnGluedInit);
+        SubscribeLocalEvent<GlueComponent, GetVerbsEvent<UtilityVerb>>(OnUtilityVerb);
         SubscribeLocalEvent<GluedComponent, GotEquippedHandEvent>(OnHandPickUp);
     }
 
     // When glue bottle is used on item it will apply the glued and unremoveable components.
-    private void OnInteract(Entity<GlueComponent> entity, ref AfterInteractEvent args)
+    private void OnInteract(EntityUid uid, GlueComponent component, ref AfterInteractEvent args)
```

Comment:
Why go back to an outdated format?

Reply:
Didn't know Entity<> was best practice since I've seen the old format in the codebase a lot more; will use the new format going forward!